packages/openai/src/openai-transcription-model.test.ts

```diff
@@ -1,13 +1,17 @@
 import { createTestServer } from '@ai-sdk/provider-utils/test';
 import { OpenAITranscriptionModel } from './openai-transcription-model';
 import { createOpenAI } from './openai-provider';
-import { readFile } from 'node:fs/promises';
-import path from 'node:path';
 
-const audioData = await readFile(path.join(__dirname, 'transcript-test.mp3'));
 const provider = createOpenAI({ apiKey: 'test-api-key' });
 const model = provider.transcription('whisper-1');
 
+const getAudioData = () =>
+  fetch(
+    'https://github.com/vercel/ai/raw/refs/heads/main/examples/ai-core/data/galileo.mp3',
+  )
```

Comment:
The tests should run offline. The file is in the repository, it can be loadedvia fs.readSync and a relative path.

Reply:
Also you can load it sync and store it in a variable here, no need async functions below (revert to `audioData`)